cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -66,6 +66,14 @@ Virtualization Validation Program (SVVP) to run Windows Server workloads.
 ** SVVP Certificate Red Hat Enterprise Linux CoreOS 8 workers are named _Red Hat OpenShift
 Container Platform 4 on RHEL CoreOS 8_.
 
+=== Web console
```

Comment:
There is already a 'web console' heading. Please remove.

Reply:
Done.